src/displayapp/screens/WatchFaceDigital.cpp

```diff
@@ -115,7 +114,15 @@ void WatchFaceDigital::Refresh() {
 
   notificationState = notificatioManager.AreNewNotificationsAvailable();
   if (notificationState.IsUpdated()) {
-    lv_label_set_text_static(notificationIcon, NotificationIcon::GetIcon(notificationState.Get()));
+    if (notificationState.Get()) {
+      lv_obj_set_style_local_text_color(notificationIcon, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(0x00FF00));
+      lv_label_set_text_static(notificationIcon, NotificationIcon::GetIcon(true));
+    } else if (notificatioManager.NbNotifications() > 0) {
```

Comment:
This notificatio typo has been fixed since this PR was made.

```suggestion
    } else if (notificationManager.NbNotifications() > 0) {
```